bench/Main.hs

```diff
@@ -1,28 +1,27 @@
 {-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE TemplateHaskell   #-}
+{-# LANGUAGE TemplateHaskell #-}
 
 module Main where
 
-import           Control.Monad.Except
-import           Data.Functor.Identity
-
-import qualified Control.Exception          as E
-import qualified Criterion.Main             as C
-import qualified Data.ByteString            as B
-import qualified Data.ByteString.Char8      as BC
-import qualified Data.FileEmbed             as FE
-import qualified Data.Text                  as T
-import qualified Data.Text.Encoding         as TE
+import Control.Exception qualified as E
+import Control.Monad.Except
+import Criterion.Main qualified as C
+import Data.ByteString qualified as B
+import Data.ByteString.Char8 qualified as BC
+import Data.FileEmbed qualified as FE
+import Data.Functor.Identity
 -- import qualified Data.Hashable                as Hash
```

Comment:
While this is getting churned let’s remove this line (unless you want to keep the PR scoped to formatting changes only).
```suggestion
```